functions/go/inflate-helm-chart/go.mod

```diff
@@ -3,8 +3,10 @@ module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/inflat
 go 1.16
 
 require (
-	github.com/imdario/mergo v0.3.12
+	github.com/imdario/mergo v0.3.5
```

Comment:
Why downgrade it?

Reply:
changed back to 3.12; no particular reason other than thats what was there before